app/src/main/kotlin/com/proxerme/app/helper/MaterialDrawerHelper.kt

```diff
@@ -133,17 +158,49 @@ class MaterialDrawerHelper(context: Activity, toolbar: Toolbar,
 
     private fun buildDrawer(context: Activity, toolbar: Toolbar, accountHeader: AccountHeader,
                             savedInstanceState: Bundle?): Drawer {
-        return DrawerBuilder(context)
+        val stickyDrawerItems = generateStickyDrawerItems()
+        this.stickyItemIds = stickyDrawerItems
+                .mapNotNull { DrawerItem.fromOrNull(it.identifier) }
+                .toTypedArray()
+
+        val builder = DrawerBuilder(context)
                 .withToolbar(toolbar)
                 .withAccountHeader(accountHeader)
                 .withDrawerItems(generateDrawerItems())
-                .withStickyDrawerItems(generateStickyDrawerItems())
+                .withStickyDrawerItems(stickyDrawerItems)
                 .withOnDrawerItemClickListener { view, id, item ->
                     onDrawerItemClick(view, id, item)
                 }
                 .withShowDrawerOnFirstLaunch(true)
-                .withActionBarDrawerToggleAnimated(true)
                 .withTranslucentStatusBar(true)
+                .withGenerateMiniDrawer(DeviceUtils.isTablet(context))
+                .withSavedInstance(savedInstanceState)
+
+        return if (DeviceUtils.isTablet(context)) builder.buildView() else builder.build()
+    }
+
+    private fun buildTabletDrawer(context: Activity, drawer: Drawer, savedInstanceState: Bundle?):
+            Pair<MiniDrawer?, Crossfader<*>?> {
+        if (DeviceUtils.isTablet(context)) {
+            val miniDrawer = drawer.miniDrawer.withIncludeSecondaryDrawerItems(true)
+            val crossfader = buildCrossfader(context, drawer, miniDrawer, savedInstanceState)
+            miniDrawer.withCrossFader(CrossfadeWrapper(crossfader))
```

Comment:
Blank line above please. I always prefer a blank line between assignments and invokations.